pkg/iris/events.go

```diff
@@ -75,3 +76,43 @@ type Log struct {
 func GenerateRunId() string {
 	return uuid.New().String()
 }
+
+type StatusItem struct {
+	ID              string      `json:"id"`
+	Scope           string      `json:"scope"`
+	Type            string      `json:"type"`
+	Source          string      `json:"source"`
+	Time            string      `json:"time"`
+	DataContentType string      `json:"datacontenttype"`
+	DataSchema      string      `json:"dataschema"`
+	Subject         string      `json:"subject"`
+	Data            interface{} `json:"data"`
+}
+
+func NewStatusEvent(configurationId string, runId string, workspaceId string, runstatus string, message string) *StatusItem {
+
+	id := fmt.Sprintf("%s", runId)
+	Subject := fmt.Sprintf("Status")
+	Type := fmt.Sprintf("leanix.vsm.item-logged.status")
+	scope := fmt.Sprintf("workspace/%s", workspaceId)
+	Source := fmt.Sprintf("kubernetes/%s#%s", configurationId, runId)
+	Time := fmt.Sprintf(time.Now().Local().String())
+	datacontenttype := fmt.Sprintf("application/json")
+	dataschema := fmt.Sprintf("/vsm-iris/schemas/operation-item/v1")
```

Comment:
some are lower case, some upper case. let's consolidate on lower case and use camelCase

Reply:
Updated to camelCase.

Reply:
But we have to keep the json attributes all lowercase as expected.